template.yaml

```diff
@@ -28,7 +28,13 @@ Globals:
     Runtime: nodejs16.x
 
 Resources:
-  HelloWorld:
+  ZendeskWebhook:
```

Comment:
Could we call this `ValidateRequestFunction` or something along those lines? Assuming that this is the first function in our flow, and it's going to validate the details in the Zendesk ticket.